scripts/ci/tags.yaml

```diff
@@ -546,3 +546,13 @@ esb:
     - nrf/subsys/mpsl/
     - nrfxlib/crypto/
     - zephyr/soc/nordic/
+
+sample_app_event_manager:
+  files:
+    - nrf/samples/app_event_manager/
+    - nrf/subsys/app_event_manager/
```

Comment:
Consider adding public and private header of app event manager (also for test). Private header contains some implementation.